app/mailers/invoice_mailer.rb

```diff
@@ -38,4 +38,12 @@ def ensure_pdf
 
     Invoices::GeneratePdfService.new(invoice:).call
   end
+
+  private
+
+  def ensure_pdf
```

Comment:
Lint/DuplicateMethods: Method `InvoiceMailer#ensure_pdf` is defined at both app/mailers/invoice_mailer.rb:36 and app/mailers/invoice_mailer.rb:44.